peregrine/resources/submission/__init__.py

```diff
@@ -12,7 +12,7 @@
 import datamodelutils.models as models
 import flask
 
-from peregrine.auth import current_user, get_read_access_projects
+from peregrine.auth import get_read_access_projects
```

Comment:
In the PR description do you mean id tokens?

Reply:
no we just want to be able to make requests without sending an Authorization header with an access token